src/transformers/models/graphormer/modeling_graphormer.py

```diff
@@ -798,9 +798,11 @@ def forward(
         attn_edge_type,
         perturb=None,
         masked_tokens=None,
-        return_dict: Optional[bool] = True,
+        return_dict: Optional[bool] = None,
```

Comment:
`PretrainedConfig` has default value `True` for `return_dict`. And in `transformers`, we usually use `return_dict: Optional[bool] = None` in `forward`, and adding one line in the beginning of `forward`
```python
return_dict = return_dict if return_dict is not None else self.config.use_return_dict
```
which is
```
return self.return_dict and not self.torchscript
```
to avoid torchscript failure.